src/reporter/github.js

```diff
@@ -114,25 +162,30 @@ export default class GitHub {
 
   async closeIssueWithCommentIfExists({ title, comment }) {
     try {
```

Comment:
I feel that only the calls to GitHub should be wrapped in a `try/catch`, as they are the only predictable points of failure.

Reply:
The behavior will almost be the same (as plugin errors won't stop the process) and if we do that we need three try catch in this function. So I'm not in favor of doing this